dragula.js

```diff
@@ -605,4 +613,66 @@ function getCoord (coord, e) {
   return host[coord];
 }
 
+function getScrollContainer(node) {
+  if (node === null) { return null; }
+  if (node.scrollHeight > node.clientHeight) {
+    return node;
+  } else {
+    if (!/(body|html)/i.test(node.parentNode.tagName)) {
```

Comment:
@nguyenj I just meant putting it in a variable like this:

```js
var rbodyhtml = /(body|html)/i
if (!rbodyhtml.test(node.parentNode.tagName)) {
  // ...
```

I avoid inline regular expressions in conditionals because they dampen readability